webknossos/webknossos/dataset/dataset.py

```diff
@@ -120,6 +120,10 @@
     + r"(?P<organization_id>[^/]*)/(?P<dataset_name>[^/]*)(/(view)?)?"
     + r"(\?token=(?P<sharing_token>[^#]*))?"
 )
+# _ZARR_DATASET_URL_REGEX = re.compile(
```

Comment:
What about this?

Reply:
This might be part of a follow up. I created an issue (#1216) for discussing the exact responsibilities for `Dataset.open()` and `Dataset.open_remote()`. Should I remove the code for now or add something like a TODO label to it?